### PR TITLE
🎨 [Frontend] Logger UX improvements

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

- [x] Wrap long texts, and when clicking on the line, extend it
- [x] Allow coping one log line
- [x] If a search is applied, download only logs corresponding to the search
- [x] Switch the log-lock's bahaviour
- [x] Clicking on the ``failed`` status chip in the node, brings you to the logger and filters that node

Logger

FailedNode

## Related issue/s

closes https://github.com/ITISFoundation/osparc-issues/issues/1487
closes https://github.com/ITISFoundation/osparc-issues/issues/1139
closes https://github.com/ITISFoundation/osparc-issues/issues/651
related to https://github.com/ITISFoundation/osparc-issues/issues/1684

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js [hunk 1]
```diff
@@ -266,6 +266,22 @@ qx.Class.define("osparc.desktop.StudyEditor", {
 
       workbench.addListener("updateStudyDocument", () => this.updateStudyDocument());
       workbench.addListener("restartAutoSaveTimer", () => this.__restartAutoSaveTimer());
+
+      // OM remove this
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js:270]: did you forget to remove this?

**Review by pcrespov — APPROVED** [review]
this is very welcomed!

**Review by sanderegg — APPROVED** [review]
love it! mesmerizing!

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js:270]: Indeed, thanks!

File: services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js [hunk 2]
```diff
@@ -266,6 +266,22 @@ qx.Class.define("osparc.desktop.StudyEditor", {
...
+      this.getStudyLogger().info(null, "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.");
+      for (i=51; i<70; i++) {
+        this.getStudyLogger().info(null, "Short log " + i);
+      }
+      /*
+      setInterval(() => {
+        this.getStudyLogger().info(null, "Short" + i);
+        i++;
+      }, 10000);
+      */
```
[the remark this answers is not in the document]
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js:284]: Yep 👍

**Review by jsaq007 — APPROVED** [review]

**Pull request merged**